### PR TITLE
Hotfix for shared gpu resources

### PR DESCRIPTION
**Pull request by llde**
Hi, using the excuse of the (luckly mild) earthquake that happened I made an important hotfix for shared gpu resouce patchset.

Basically  32 bit thunk conversion were converting VkCommandBuffer to native handles in every case they were involved in structure conversion bypassing original conversion functions and unwrapping needs (probably becouse they took the need_conversion True path as they are dispatchable), so they were unwrapped also when Thunk Type wasn't Public. This error depended on make_vulkan script. 64 bit thunks weren't affected.
This was causing an assertion in 32 bit application using DXVK.  Added a comment inside make_vulkan.
Doesn't seems other error of this sort are present inside the thunks.

Also added the fences patchset for the 7.20 patchset I forgot to include and to use from the script.

Asseghimma

### CODE REVIEW

File: wine-tkg-git/wine-tkg-patches/proton/shared-gpu-resources/sharedgpures-fences.patch [hunk 1]
```diff
@@ -131,7 +131,26 @@ index beb93d87ad2..01b2f048b39 100755
...
              LOGGER.error("Unhandled native handle for: {0}".format(self.name))
          return None
-@@ -1893,7 +1926,7 @@ class VkParam(VkVariable):
+@@ -1480,7 +1513,8 @@ class VkMember(VkVariable):
+                 else:
+                     # Nothing needed this yet.
+                     LOGGER.warn("TODO: implement copying of static array for {0}.{1}".format(self.type, self.name))
+-            elif self.is_handle() and self.needs_unwrapping():
++            elif self.is_handle() and unwrap: ## Was self.need_unwrapping(). VkCommandBuffer command buffer wans't supposed to be converted to native handle from VkCommandBufferSubmitInfo conversion.
++                 #This was hitten in 32 bit conversion probably becouse it's a dispatchable type so conv == True. Force unwrap usage instead (unwrap is True for  Public Thunks, so direct call to uni side without PE implementation)
```
Comment (Tk-Glitch) [inline wine-tkg-git/wine-tkg-patches/proton/shared-gpu-resources/sharedgpures-fences.patch:140]: Do we keep the typos for fun (they are funny yet don't make the sentence unreadable imo)?

**Comment by Tk-Glitch** [comment]
Aside from the couple typos in that comment I'll wait for an answer from you, I approve. Thanks @llde :frog: :heart:

File: wine-tkg-git/wine-tkg-patches/proton/shared-gpu-resources/sharedgpures-fences.patch (hunk 1 above)
Comment (llde) [reply wine-tkg-git/wine-tkg-patches/proton/shared-gpu-resources/sharedgpures-fences.patch]: @Tk-Glitch fixed typos. (Kept hitten that while is considered archaic is correct)

**Comment by Tk-Glitch** [comment]
Less funny for sure. Thanks again @llde !

**Pull request merged**